python/lsst/summit/utils/imageExaminer.py

```diff
@@ -584,8 +589,12 @@ def plotFullExp(self, ax: matplotlib.axes.Axes | None = None) -> None:
             plotDirect = True
 
         imData = quickSmooth(self.exp.image.array, 2.5)
-        vmin = np.percentile(imData, 10)
-        vmax = np.percentile(imData, 99.9)
+        # np.percentile returns a float if imData is a 1D array,
+        # but it can return a numpy array if imData is a 2D array
+        # or higher. If imData is a 2D array, vmin and vmax will
+        # be arrays, which is not what LogNorm expects.
+        vmin = np.percentile(imData, 10).item()
+        vmax = np.percentile(imData, 99.9).item()
```

Comment:
but `imData` here _is_ 2d, and it's giving a scalar already, so that's not necessary for the code to work - it will only give you an array if you specify `axis=0` (or `1`), so I don't love this. Is it really giving errors? I didn't know about calling `.item()` so I find that a little confusing and would prefer to avoid that if it's possible.

Reply:
Problem is in:
LogNorm(vmin=vmin, vmax=vmax)
Which only accepts vmin and vmax to be a float,